packages/components/.storybook/main.js

```diff
@@ -7,4 +7,16 @@ module.exports = {
 		check: false,
 		reactDocgen: false,
 	},
+	webpackFinal: async ( config, { configType } ) => {
+		config.resolve.alias = {
+			...config.resolve.alias,
+			// Storybook does not support Emotion 11, so resolve it manually.
+			// TODO: Remove once Storybook supports Emotion 11.
+			'@emotion/styled': path.join( process.cwd(), 'node_modules/@emotion/styled' ),
```

Comment:
`path.join( process.cwd(), 'node_modules/@emotion/styled' )` seems a bit weak: if the process to launch storybook is run from a different directory, it will generate the wrong path. Plus if `@emotion` get hoisted differently it may point to the wrong version.

What about using require.resolve to find `@emotion` path?

Reply:
I can look into that :+1:

Reply:
yep, this worked!